src/simulator.ts

```diff
@@ -21,7 +21,7 @@ export class Simulator {
   public static getInstance(context?: vscode.ExtensionContext) {
     if (!Simulator.instance) {
       if (!context) {
-        vscode.window.showErrorMessage("Cannot create Simulator instance.");
+        vscode.window.showErrorMessage("Cannot create Send D2C Messages webview instance.");
```

Comment:
remove "instance"?

Reply:
Actually it's not creating the webview. I will update the error to: Cannot initialize Send D2C Messages webview resources.